src/localization/ru/lessons/regexForSeo.json

```diff
@@ -1,49 +1,49 @@
+  "lesson.regexForSeo.or.title": "Оператор ИЛИ `|`",
+  "lesson.regexForSeo.or.description": "Мы используем оператор ИЛИ `|`, чтобы выбрать несколько выражений. Например, выберите текст, содержащий `seo` и `regex`, написав два слова, разделяя их оператором ИЛИ `|`.",
 
-  "lesson.regexForSeo.optional.title": "Optional Operator `?`",
-  "lesson.regexForSeo.optional.description": "We use it to express that an expression or character is optional. For example, we want to filter text containing both `http` and `https`. We can use the Or operator `|` for this operation, but there is a better way for these two almost identical words.\\n\\nIn this example, the letter `s` is optional. Instead, add a question mark `?` to the end of the letter `s`.",
+  "lesson.regexForSeo.optional.title": "Опциональный оператор `?`",
+  "lesson.regexForSeo.optional.description": "Мы используем его, чтобы указать, что выражение или символ являются необязательными. Например, мы хотим выбрать текст, содержащий одновременно `http` и `https`. Мы можем использовать оператор ИЛИ `|` для этой операции, но есть способ лучше для этих двух почти одинаковых слов.\\n\\nВ этом примере, буква `s` необязательна. Вместо этого, добавьте вопросительный знак `?` после буквы `s`.",
 
-  "lesson.regexForSeo.any.title": "Anything `.*`",
-  "lesson.regexForSeo.any.description": "In regex, the period `.` can match anything, including spaces. But this match is only for a single character where it is used. The asterisk character `*` means that the preceding item does not exist at all or occurs more than once in the text. When the two are used together, it means that there can be an element of unlimited length where used.\\n\\nBelow are various keywords. Add `.*` to the end of the regex to write the expression that starts with `how to write` and matches everything after it.",
+  "lesson.regexForSeo.any.title": "Все символы `.*`",
+  "lesson.regexForSeo.any.description": "В регулярном выражении, точка `.` может соответствовать всем символам, включая пробелы. Но она соответствует только одному символу, где она была использована. Символ звездочки `*` означает, что предшествующий символ отсутствует или появляется в тексте больше одного раза. Когда они используются вместе, это означает, что элемент неограниченной длины может находиться там, где они использованы.\\n\\nНиже представлены различные ключевые слова. Добавьте `.*` в конец регулярного выражения, чтобы написать выражение, которое начинается с `how to write` и соответствует всему после него.",
 
-  "lesson.regexForSeo.contains.title": "Contains",
-  "lesson.regexForSeo.contains.description": "To filter the text that contains the words or phrases we're looking for, we write the word we're looking for between our two phrases `.*` that match everything. For example, type `.*buy.*` to filter out what includes `buy`.",
+  "lesson.regexForSeo.contains.title": "Содержание",
```

Comment:
Could this translation be more accurate?

`Contains` -> `Содержит`

Reply:
I would not say that it is good translation. In the title, the verb will be without the noun, and it will look in Russian abrupt (like machine translation), like "contains (содержит)" what?. 
`Содержание` can be more precise title in this case, since it will be clear that it is containment of something (the word or phrase).